Src/Sarif.PatternMatcher.Cli/Program.cs

```diff
@@ -20,6 +22,15 @@ internal static int Main(string[] args)
         {
             Console.OutputEncoding = Encoding.UTF8;
 
+            args = EntryPointUtilities.GenerateArguments(args, FileSystem, new EnvironmentVariables());
```

Comment:
>GenerateArguments [](http://example.com/codeflow?start=39&length=17)

This is the essential change. This driver helper processes all args by expanding any of them that reference a response file.